docs/contributing.rst

```diff
@@ -1,61 +1,16 @@
 .. _aiohttp-contributing:
 
-Contributing
-============
+.. include:: ../CONTRIBUTING.rst
+   :end-before: export-point-instructions-for-contributors
 
 (:doc:`contributing-admins`)
```

Comment:
Why not keep this in the same place: at the top of the document?

https://aiohttp--9742.org.readthedocs.build/en/9742/contributing.html#instructions-for-contributors